platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -190,44 +190,9 @@ MGL_EXPORT
 - (void)cancel;
 
 /**
- The zoom level.
- 
- The default zoom level is 0. If this property is non-zero and the camera
- property is non-nil, the camera’s altitude is ignored in favor of this
- property’s value.
- */
-@property (nonatomic) double zoomLevel;
```

Comment:
Removing this public symbol would technically be a backwards-incompatible change. On the other hand, these properties are have no effect and are never set, so I guess we can assume no one is using them?

Reply:
This was released since 3.7.0 I think it's use is not spread. Since this is backwards-incompatible we may want to release it as part of 4.0 and not in a patch.

Reply:
That sounds reasonable to me. Alternatively, if a patch release is going out the door anyways, we can mark these methods as deprecated for now and remove them when v4.0.0 comes out.